evalml/pipelines/components/transformers/preprocessing/datetime_featurizer.py

```diff
@@ -77,8 +76,7 @@ def __init__(self, features_to_extract=None, encode_as_categories=False, random_
 
     def fit(self, X, y=None):
         X = _convert_to_woodwork_structure(X)
-        X = _convert_woodwork_types_wrapper(X.to_dataframe())
-        self._date_time_col_names = X.select_dtypes(include=datetime_dtypes).columns
+        self._date_time_col_names = X.select(include=["datetime"]).columns
```

Comment:
Can't do `X.select('datetime').columns` ? Just curious because we're doing that elsewhere for numeric